scripts/rtl872x.tcl

```diff
@@ -107,7 +108,8 @@ proc openocd_print_memory {address length} {
     init
     reset halt
     set value ""
-    mem2array value 8 $address $length
+		set value [read_memory $address 8 $length]
```

Comment:
Nit: spaces vs tabs. Same above.

Reply:
hehe fixed! My vim is not configured well hehe 😅